README.md

```diff
@@ -1,103 +1,130 @@
 # Orbs Staking Wallet (Tetra)
+
 Web client that enables users to take part in the ORBS pos staking process.
 All contract interactions are done with the [orbs-pos-data](https://github.com/orbs-network/orbs-ethereum-contracts-v1) library.
 
 ## Test
 
-* For unit tests, run in terminal in root project directory:
+- For unit tests, run in terminal in root project directory:
 
   ```
   npm run test
-  ``` 
+  ```
 
 ### Running - Local ethereum network - synthetic committee & stake
 
-* Start local ethereum network (Ganache)
-   ```
-    cd ./ganach-env
-    npm install 
-    npm run up-dev
-   ```
-  
-* Start local dev server (from the root folder of the project)
+- Start local ethereum network (Ganache)
+
+  ```
+   cd ./ganach-env
+   npm install
+   npm run up-dev
+  ```
+
+- Start local dev server (from the root folder of the project)
 
   ```
   npm run dev
-  ```    
-* [Setup Metamask wallet](#test-tetra)
+  ```
+
+- Start local dev server to test mobile wallet (from the root folder of the project)
+  install serve-js - npm install serve
```

Comment:
we can include these packages under the "dev" section in package.json to remove this installation step - especially since these will be committed next time someone commits.